QuizAppNew/View/HomeView.swift

```diff
@@ -8,13 +8,79 @@
 import SwiftUI
 
 struct HomeView: View {
+    
+    @State var show = false
+ // Storing level for fetching questions....
+    @State var set = "Round_1"
+    
+    // for analytices.....
```

Comment:
Why correct, wrong, answer parameter why need it. You are only responsible for making UI design.

Reply:
correct wrong selected variable is removed